lib/packages/util/package.json

```diff
@@ -57,6 +57,8 @@
   },
   "gitHead": "9b5a4e07616d783e85bcae6c21b72f50db783f48",
   "dependencies": {
+    "@types/big.js": "^4.0.5",
+    "big.js": "^5.2.2",
```

Comment:
Needed for __much__ better precision. Once, we had `bn.js`, but this one is much more lightweight...